### ISSUE TITLE
jinja2 ImportError prevents web app from running

### ISSUE THREAD

**Issue by clegaspi** [clegaspi/saml_reader#74]
When installing the package from PyPI, there are some packages that had breaking changes which cause this error to crop up:

```
ImportError: cannot import name 'escape' from 'jinja2' 
```

To patch, run:

```
pip install --upgrade python3-saml
pip install --force-reinstall defusedxml==0.6.0
pip install lxml==4.6.5 dash-extensions==0.0.71 Flask==1.1.2 Jinja2==3.0.1 itsdangerous==2.0.1 Werkzeug==2.0.1
```

**Comment by clegaspi** [comment]
Even with #75, things continue to not function correctly as the internal packages for dash and Flask get updated. The app needs to be made compatible with the newer versions of these packages.

**Comment by clegaspi** [comment]
Fixed this in 0.0.6.
